0-bootstrap/README.md

```diff
@@ -108,27 +108,21 @@ your current Jenkins manager (master) environment.
 ## Deploying with Cloud Build
 
 1. Go to the `0-bootstrap` folder.
-1. Copy the `tfvars` file:
-   ```
-   cp terraform.example.tfvars terraform.tfvars
-   ```
-1. Update the `terraform.tfvars` file with values from your environment.
+1. Rename `terraform.example.tfvars` to `terraform.tfvars` and update the file with values from your environment.
```

Comment:
Can we include the command to do this? i.e `mv terraform.example.tfvars terraform.tfvars` so that a user can easily copy and paste if they wish to do that.

Reply:
Added mv command